### ISSUE TITLE
Allow multiple ref on power lines

### ISSUE THREAD

**Issue by wolfy1339** [openinframap/openinframap#127]
Currently `ref`s are only rendered when there is only one, when there are multiple it doesn't render.
This situation arises when there are multiple power circuits on the same towers.
It should be as simple as splitting the `ref` property by `;` and putting each ref on it's own line within the label, like OSM does with highway numbers

### Screenshot
![image](https://user-images.githubusercontent.com/4595477/147371123-f574c446-3d70-4a1e-a34c-1cdfd061b4c4.png)
### Link to where this issue occurs on OpenInfraMap
https://openinframap.org/#15.05/46.17554/-70.6596
### Link to the corresponding entity on OpenStreetMap
https://www.openstreetmap.org/way/979914198#map=17/46.17661/-70.66375

**Comment by flacombe** [comment]
It should be said that if those refs are actually related to power circuits, they should go in relations involving the power line.
Like frequency and operating voltages.

We're still waiting this proposal be finished : https://wiki.openstreetmap.org/wiki/Proposed_features/Power_routing_proposal

Intermediate scheme has been rolled out in France, waiting for the final answer on this particular topic : https://www.openstreetmap.org/user/InfosReseaux/diary/392777
These two relations involve similar power lines, with respective ref:FR:RTE:
* https://www.openstreetmap.org/relation/5995928
* https://www.openstreetmap.org/relation/5995937

**Comment by russss** [comment]
Yes, I will definitely take another look at this once the routing proposal is finalised.

Incidentally in Great Britain, the ref does refer to the actual power line, not to the circuits (which only have names, not refs).

**Comment by flacombe** [comment]
It is good to have different references, respectively to physical lines and circuits over them.  
It ease any conflation or data linkage.
